ProjFS.Mac/PrjFSKextTests/VirtualizationRootsTests.mm

```diff
@@ -264,4 +271,116 @@ - (void)testRegisterProviderForPath_ExistingRecycledRoot
 
 }
 
+- (void)testFindForVnode_FileInRoot
+{
+    const char* repoPath = "/Users/test/code/Repo";
+    const char* filePath = "/Users/test/code/Repo/file";
```

Comment:
Worth adding a case for a file with a deeper path?

Reply:
I feel like the other tests, such as `testFindForVnode_DirectoryInUndetectedRoot` already cover that case.